NodeJS/Project/test/test.js

```diff
@@ -0,0 +1,67 @@
+const expect = require('chai').expect;
+const stack = require('../lib/stack.js');
+
+describe('stack#pop', function (){
```

Comment:
In order to get everything organized, you can also stack the more than one `describe` function. For example:
```javascript
describe('stack', function (){
  describe('pop', function (){
    it(...)
    it(...)
    it(...)
....
  })
  describe('push', function (){
    it(...)
    it(...)
    it(...)
....
  })
})
```